pyanaconda/startup_utils.py

```diff
@@ -71,8 +92,365 @@ def get_anaconda_version_string():
         # Ignore pylint not finding the version module, since thanks to automake
         # there's a good chance that version.py is not in the same directory as
         # the rest of pyanaconda.
-        from pyanaconda import version # pylint: disable=no-name-in-module
+        from pyanaconda import version  # pylint: disable=no-name-in-module
```

Comment:
That's actually deliberate - [PEP8 specifies](https://www.python.org/dev/peps/pep-0008/#inline-comments) that incline comments should be prefixed by at least two spaces:

> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.  

It's a minor thing but I'm trying to be consistent when at it. :)